frontend/mulighetsrommet-veileder-flate/src/components/FilterLoader.tsx

```diff
@@ -0,0 +1,9 @@
+import { Loader } from "@navikt/ds-react";
+
+export function FilterLoader() {
+  return (
+    <div style={{ display: "block", margin: "0 auto", textAlign: "center" }}>
+      <Loader size="xlarge" title="Laster tiltaksgjennomføring..." />
```

Comment:
Komponentnavn rimer ikke helt med denne tittelen. Kunne kanskje kalt den for `TiltakLoader` og endret tekst til `Laster tiltak...`? Da passer vel flertallsformet både for både oversikt- og detaljvisning også

Reply:
Yes - enig! ✅